src/PhpSpec/CodeAnalysis/StaticRejectingNamespaceResolver.php

```diff
@@ -43,7 +43,15 @@ public function resolve($typeAlias)
      */
     private function guardScalarTypeHints($typeAlias)
     {
-        if (in_array($typeAlias, array('int', 'float', 'string', 'bool'))) {
+        $scalarTypes = [
```

Comment:
These aren't really scalar types any more, I guess? Can you rename the private method and this local variable please?